src/client/apps/edit/components/content/sections/related_articles/index.tsx

```diff
@@ -41,9 +51,13 @@ export class RelatedArticles extends Component {
         .query({ query: RelatedArticleQuery(idsToFetch) })
         .end((err, res) => {
           if (err) {
-            console.error(err)
+            new Error(err)
+          }
+          const articles =
+            res && res.body && res.body.data && res.body.data.articles
```

Comment:
Do we use lodash in this repo? We could probably use [get](https://lodash.com/docs/4.17.14#get) here if so.

```js
const article = get(res, "res.body.data.articles")
```

But that's minor and not at all a blocker.